tests/test_launcher.py

```diff
@@ -204,6 +207,67 @@ def test_watchdog_launch(monkeypatch):
     pyfluent.launch_fluent(start_watchdog=True)
 
 
+def test_fluent_launchers():
+    if not check_docker_support() and not pypim.is_configured():
+        standalone_meshing_launcher = create_launcher("standalone", mode="meshing")
+        standalone_meshing_session = standalone_meshing_launcher()
+        assert standalone_meshing_session
+        assert standalone_meshing_launcher.start_transcript is True
```

Comment:
@seanpearsonuk Testing standalone launcher, session and default value of `start_transcript` attribute.